### PR TITLE
xGen Conference

### PR DESCRIPTION
**Pull request by AbbasSarwar**
### Feature
* Responsive <768px> 
* About & Home Page 
* Fonts & Colors (Lato)
* (Images) background own.
* Menu clickable
* Dynamic Speakers section

Project Documentation: https://www.loom.com/share/057f9aafad5a4bcba6d3e82c8f453b40

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,15 +5,230 @@
...
+        <div class="footer-logo">
+            <h5>CC</h5>
+            <h4>creative<br> commons united</h4>
+        </div>
+        <div class="footer-details">
+            <h5>2015 Creative Commons United. Some Rights Reserved.</h5>
+            <p>Conferences are gatherings where professionals with shared interests connect, learn, and collaborate on new ideas and developments, providing a platform for networking and knowledge sharing.</p>
+        </div>
+    </section>
+    <section class="about" id="about">
```
Comment (souad988) [inline index.html]: Great work was done so far 💪, But in the requirement this section is meant to be a separate page the about page is missing in your project , please make sure to follow the requirements step by step

File: index.html [hunk 2]
```diff
@@ -5,15 +5,230 @@
...
+        <nav class="menu-container">
+            <div class="logo">
+                <a href=""><img src="image-resources/xgen-conference-logo__2.png" alt=""></a>
+            </div>
+            <div class="hamburger border">
+                <span class="bar bar1"></span>
+                <span class="bar bar2"></span>
+                <span class="bar bar3"></span>
+                 <ul class="menus">
+                    <li><a href="#about" class="text-decoration-none p-2">About</a></li>
```
Comment (souad988) [inline index.html]: Kindly make this link redirect user to about.html page <a href="./about.html">About</a> as it is in the requirements

File: index.html [hunk 3]
```diff
@@ -5,15 +5,230 @@
...
+    <script src="js/speakers.js" type="module"></script>
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css" integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
     <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/js/bootstrap.bundle.min.js" integrity="sha384-w76AqPfDkMBDXo30jS1Sgez6pr3x5MlQ1ZAGC+nuZB+EYdgRZgiwxhTBTkF7CXvN" crossorigin="anonymous" type="module"></script>
     <!--End here-->
     <title>XGen Conference</title>
 </head>
 <body>
-    
+    <header>
```
Comment (souad988) [inline index.html:24]: Good job your header looks good , but it is missing the dark top part with social media links 

top header missing

File: index.html [hunk 4]
```diff
@@ -5,15 +5,230 @@
...
+                    <li><a href="#join" class="text-decoration-none">Join</a></li>
+                    <li><a href="#sponsor" class="text-decoration-none">Sponsor</a></li>
+                    <li><a href="#news" class="text-decoration-none">News</a></li>
+                    <li><a href="#cc" class="text-decoration-none p-2">CC Campaign</a></li>
+                </ul>
+            </div>
+        </nav>
+        <section class="top-heading mt-5">
+            <div class="main-head">
+                <p>"Hello !Sharing World"</p>
```
Comment (souad988) [inline index.html]: Please always use hx tags for titles as p tag is used for paragraphs this is required for semantic reasons

File: index.html [hunk 5]
```diff
@@ -5,15 +5,230 @@
...
+                    <li><a href="#about" class="text-decoration-none p-2">About</a></li>
+                    <li><a href="#program" class="text-decoration-none">Program</a></li>
+                    <li><a href="#join" class="text-decoration-none">Join</a></li>
+                    <li><a href="#sponsor" class="text-decoration-none">Sponsor</a></li>
+                    <li><a href="#news" class="text-decoration-none">News</a></li>
+                    <li><a href="#cc" class="text-decoration-none p-2">CC Campaign</a></li>
+                </ul>
+            </div>
+        </nav>
+        <section class="top-heading mt-5">
```
Comment (souad988) [inline index.html:52]: Kindly try to stick as much as possible to the given design in terms of spacing text size and weight 

mobile conference student

mobile conference

File: index.html [hunk 6]
```diff
@@ -5,15 +5,230 @@
...
+    <section class="footer footer-hide">
+        <div class="footer-logo">
+            <h5>CC</h5>
+            <h4>creative<br> commons united</h4>
+        </div>
+        <div class="footer-details">
+            <h5>2015 Creative Commons United. Some Rights Reserved.</h5>
+            <p>Conferences are gatherings where professionals with shared interests connect, learn, and collaborate on new ideas and developments, providing a platform for networking and knowledge sharing.</p>
+        </div>
+    </section>
```
Comment (souad988) [inline index.html:169]: good job in the footer as well 😊💪 , but it should be the last element in your page as it is a footer as I told you before put all about content in a separate page and don't forget to make the footer in about page look like the given template 

**home footer**
home footer

**about footer**
about footer

File: index.html [hunk 7]
```diff
@@ -5,15 +5,230 @@
...
+                    <p>get opportunities to network with CC affiliates around the world, also after the summit.</p>
+                </div>
+            </div>
+        </div>
+        <div class="btn-container" id="join">
+            <button class="btn-main">Join CC Global Summit 2015</button>
+            <a href="#" class="btn-main-desk">SEE THE WHOLE PROGRAM</a>
+        </div>
+    </section>
+    <section class="speakers">
```
Comment (souad988) [inline index.html:125]: This section is displaying content as expected and expands on mobile 👍💪✔, just fix the margin and add the missing divider also if the title is not provided don't display the empty quotes please do your best to replicate the give design 

speakers section

student speakers section

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @AbbasSarwar , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


212385281-4797c58a-d358-423a-aa7c-832c22203865


## Highlights
✔Good use of github workflows
✔Good PR description
✔No linter errors
✔Good description of the project in the video it contains a demo of different parts of the project also it highlights parts of code 

## Required Changes ♻️
_Check the comments under the review._
_ Also don't forget to redo your video after making the changes as the footer should appear in the bottom and the about section should be a different page , you have done great just fix this small issues and you will be good to go to next module   
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+        <div class="partner-details">
+            <ul>
+                <li><a href="#">mozilla</a></li>
+                <li><a href="#">Google</a></li>
+                <li><a href="#" class="font-weight-bold">NAVER</a></li>
+                <li><a href="#">daumkakao</a></li>
+                <li><a href="#"><i class="fa-brands fa-airbnb"></i> airbnb</a></li>
+            </ul>
+        </div>
+    </section>
```
Comment (franclobo) [inline about.html]: - [ ] The footer on the about page is the same as that on the home page, please 🙏🏽  correct the footer to look similar to the original design. The partner section is ok on the mobile version, but you missed the footer. 👇🏽 

| Desktop version | Mobile version |
|---|---|
| image | image |
| image | image |

File: about.html [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+                <span class="bar bar2"></span>
+                <span class="bar bar3"></span>
+                 <ul class="menus">
+                    <li><a href="about.html" class="text-decoration-none p-2">About</a></li>
+                    <li><a href="index.html#program" class="text-decoration-none">Program</a></li>
+                    <li><a href="index.html#join" class="text-decoration-none">Join</a></li>
+                    <li><a href="index.html#sponsor" class="text-decoration-none">Sponsor</a></li>
+                    <li><a href="about.html#news" class="text-decoration-none">News</a></li>
+                    <li><a href="about.html#cc" class="text-decoration-none p-2">CC Campaign</a></li>
+                </ul>
```
Comment (franclobo) [inline about.html]: - [ ] Please 🙏🏽 add the home link on the mobile menu to be able to return to the home page for a better UX.

File: index.html [hunk 10]
```diff
@@ -5,15 +5,170 @@
...
+                <h1>2023</h1>
+            </div>
+            <div class="main-box">
+                <p>Join the World Conference to explore global issues through talks, panels, and workshops. Collaborate with experts and become part of the solution through meaningful dialogue.</p>
+            </div>
+            <div class="main-date">
+                <h2>2022.1.15(MON) ~ 16(FRI)</h2>
+            </div>
+            <div class="main-center">
+                <p>@ United states of America, Science Center and more</p>
```
Comment (franclobo) [inline index.html:64]: - [ ] This text is overlapping the next section. As requested by the previous code reviewer please 🙏🏽 try to match as much as possible with the original design, kindly add a space between the introduction and the main program for a better user interface.

image

File: index.html [hunk 11]
```diff
@@ -5,15 +5,170 @@
...
+                    <li><a href="about.html#cc" class="text-decoration-none p-2">CC Campaign</a></li>
+                </ul>
+            </div>
+        </nav>
+        <section class="top-heading mt-5">
+            <div class="main-head">
+                <h6>"Hello !Sharing World"</h6>
+                <h1 >GLOBAL DIALOGUE FOR</h1>
+                <h1 >SOLUTIONS SUMMIT</h1>
+                <h1>2023</h1>
```
Comment (franclobo) [inline index.html]: - [ ] I recommend using just one h1 tag for the title and editing the width and margins on the CSS file to be similar to the original design as requested by the previous code reviewer 👇🏽 

| Original design | Your design |
|---|---|
| image | image |

File: index.html [hunk 12]
```diff
@@ -5,15 +5,170 @@
...
+                <div class="main-paragraph ">
+                    <p>get opportunities to network with CC affiliates around the world, also after the summit.</p>
+                </div>
+            </div>
+        </div>
+        <div class="btn-container" id="join">
+            <button class="btn-main">Join CC Global Summit 2015</button>
+            <a href="#" class="btn-main-desk">SEE THE WHOLE PROGRAM</a>
+        </div>
+    </section>
```
Comment (franclobo) [inline index.html:124]: - [ ] The title is overlapping with the description, please 🙏🏽 correct this behavior for a better UI. 👇🏽 

image

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @AbbasSarwar, 

# ♻️ Required Changes ♻️

soClose

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️
- Good topic 💯  

## Suggested changes
_Check the comments under the review._

You should consider:

- [ ] After submitting your project for reviewing please 🙏🏽 stop doing changes, I noticed you were editing your project while I was reviewing it 👇🏽 

image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: css/about.css [hunk 13]
```diff
@@ -0,0 +1,163 @@
...
+    background-image: linear-gradient(to right, #ec5242, #ec5242, #272a31, #ec5242, #ec5242, #ec5242);
+    /* stylelint-disable */
+    -webkit-background-clip: text;
+    -moz-background-clip: text;
+    background-clip: text;
+    /* stylelint-enable */
+    letter-spacing: 5px;
+    font-weight: 900;
+    color: transparent;
+    font-size: 24px;
```
Comment (binhussen) [inline css/about.css]: - Great Job for fixing most request changes, you missed one thing the title `letter-spacing`, `font-size`, and `font-style`...  are different from the given design for both home and about page, please fix it as the previous reviewer suggested you

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|||


😎️

🤝️

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @AbbasSarwar , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @AbbasSarwar 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - All linter checks passed ✔️
 - Descriptive pull request ✔️
 - Used GitHub flow ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**